tests/system/verbs/catkin_build/test_build.py

```diff
@@ -93,9 +92,35 @@ def test_build_dry_run():
                 assert not os.path.exists('devel')
 
 
-def test_build_all_isolated():
-    """Test building all packages in an isolated workspace"""
-    pass  # TODO: Implement test
+def test_build_all_isolate_install():
+    """Test building dependent catkin packages with isolated installspace."""
+    with redirected_stdio() as (out, err):
+        with workspace_factory() as wf:
+            n_pkgs = create_tree_workspace(wf, 'catkin', 2)
```

Comment:
It's not possible to test CMake packages in this way, since the simplified factory-generated ones don't include CMake modules which make them findable by dependencies.